testing/containers/ceph/Dockerfile

```diff
@@ -23,6 +23,7 @@ ENV PATH="${PATH}:/opt/go/bin"
 ENV GOROOT=/opt/go
 ENV GO111MODULE=on
 ENV GOPATH /go
+ENV GODEBUG=cgocheck=2
```

Comment:
I really appreciate comments about uncommon options. Maybe a link in the commit message to the docs? Not required, but in case you want/need to update the PR, think about it.

Reply:
+1 to a comment. Also, it may be better to have this particular env var in the test script, rather than in the container image. I personally sometimes use the container and the embedded ceph cluster for trying things out and I don't know if cgocheck=2 has any other side effects (I would think so, because Go doesn't have it enabled all the time).

Reply:
Let me remove the commit from this PR, I added it only to "prove" that it avoids the error. We have a separate PR #411 for this anyway, that we can merge, after this here has been merged.